lib/txgh/config/providers/git_provider.rb

```diff
@@ -36,7 +36,7 @@ def config
         def download
           github_repo.api.download(github_repo.name, payload, ref)
         rescue Octokit::NotFound
-          raise ConfigNotFoundError, "Config file #{payload} not found in #{ref}"
+          raise GitConfigNotFoundError, "Config file #{payload} not found in #{ref}"
```

Comment:
Does this not need `Txgh::`? Seems like it's defined in the same place as the other errors.